src/main/resources/data/po-lines/38434-1_pending_fomat-other.json

```diff
@@ -62,6 +59,14 @@
   "orderFormat": "Other",
   "owner": "Monographs",
   "paymentStatus": "Pending",
+  "physical": {
+    "materialSupplier": "62ce93b0-c675-43d8-9d43-a4ee7177c8de",
+    "receiptDue": null,
```

Comment:
I think we should avoid null values in sample data. Please fill `receiptDue` with valid date